webapp/src/ts/app.component.ts

```diff
@@ -646,6 +650,23 @@ export class AppComponent implements OnInit, AfterViewInit {
     });
   }
 
+  private checkBrowserCompatibility(): void {
+    if (!this.isCompatibleChromeBrowser()) {
+      this.modalService.show(BrowserCompatibilityComponent);
+    }
+  }
+
+  private isCompatibleChromeBrowser(): boolean {
```

Comment:
Is browserDectorService responsibility for checking the browser? If yes, then you can move this function to that service.